playground/buddike/lab1/file_processor.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+
+def readfilecontent(path):
+    print("Reading file content from " + path)
+
+    if os.path.exists(path) == 0:
+        print("The file does not exist")
+    else:
+        file = open(path,"r")
```

Comment:
awesome man.. 
there is a cool syntax for opening and closing the file object.. its like a using statement which closes/disposes the resource 
try this
```suggestion
       with open(path,"r") as file:
          for line in file:
             print(line)
```

Reply:
excellant, will use it